android/src/main/java/com/android/i18n/addressinput/autocomplete/gmscore/AddressAutocompleteApiImpl.java

```diff
@@ -0,0 +1,107 @@
+package com.android.i18n.addressinput.autocomplete.gmscore;
```

Comment:
This file has no license and references gmscore, I wonder if it was meant to be included? Can you check?